### DISCUSSION TITLE
when will ebpf do a v0.17.0 release cut?

### DISCUSSION THREAD

**Discussion by pacoxu** [cilium/ebpf#1596] [Q&A] [answered]
What is the typical release cycle of cilium/ebpf like? Are there any references or introductions you can recommend?

**Comment by ti-mo** [comment, marked as answer]
There's no set release cadence. It really depends on whether there are critical bugs to be fixed, changes in the upstream kernel that need to be addressed, etc. We're looking at one more major release before the end of the year.

**Reply by pacoxu** [reply]
Thanks for the information.
